promql/promqltest/testdata/aggregators.test

```diff
@@ -235,21 +235,25 @@ eval instant at 0m max(http_requests)
   {} 4
 
 # The histogram is ignored here so the result doesn't change but it has an info annotation now.
-eval_info instant at 0m max({job="api-server"})
+eval instant at 0m max({job="api-server"})
+  expect info
   {} 4
 
 # The histogram is ignored here so there is no result but it has an info annotation now.
-eval_info instant at 0m max(http_requests_histogram)
+eval instant at 0m max(http_requests_histogram)
+  expect info
 
 eval instant at 0m min(http_requests)
   {} 1
 
 # The histogram is ignored here so the result doesn't change but it has an info annotation now.
-eval_info instant at 0m min({job="api-server"})
+eval instant at 0m min({job="api-server"})
+  expect info
   {} 1
 
 # The histogram is ignored here so there is no result but it has an info annotation now.
-eval_info instant at 0m min(http_requests_histogram)
+eval instant at 0m min(http_requests_histogram)
+  expect info
 
 eval instant at 0m max by (group) (http_requests)
```

Comment:
Let's add an `expect no_info` here and to the next test to emphasize the difference to the tests above.